app/src/main/java/com/mybus/fragment/BusRouteFragment.java

```diff
@@ -21,14 +22,14 @@
  */
 public class BusRouteFragment extends Fragment {
 
-    @Bind(R.id.stop_origin_name)
```

Comment:
Why you renamed these vars? these textviews, "represent origin/destination" bus stop address.